modules/core/shared/src/main/scala/TraceValue.scala

```diff
@@ -17,17 +17,17 @@ object TraceValue {
   implicit def viaTraceableValue[A: TraceableValue](a: A): TraceValue =
     TraceableValue[A].toTraceValue(a)
 
-  @deprecated("use toTraceValue", "0.3.0")
+  @deprecated("use .viaTraceableValue(TraceableValue)", "0.3.0")
```

Comment:
I think this is a bit clearer, wdyt?
```suggestion
  @deprecated("use `viaTraceableValue(value)`", "0.3.0")
```